sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryParsingErrors.scala

```diff
@@ -369,13 +369,11 @@ object QueryParsingErrors extends QueryErrorsBase {
     new ParseException(errorClass = "DUPLICATE_KEY", messageParameters = Array(toSQLId(key)), ctx)
   }
 
-  def unexpectedFomatForSetConfigurationError(ctx: ParserRuleContext): Throwable = {
+  def unexpectedFormatForSetConfigurationError(ctx: ParserRuleContext): Throwable = {
     new ParseException(
-      s"""
-         |Expected format is 'SET', 'SET key', or 'SET key=value'. If you want to include
+      s"""Expected format is 'SET', 'SET key', or 'SET key=value'. If you want to include
```

Comment:
This is definitely an improvement over the previous state but I think we can go a bit further:
- The string interpolation is not needed as there is no outer val/var used inside.
- I do not get why replaceAll was used in the first place but a simple string concat would do especially as there is no `"` within the content.

So I would just simply:
```
"Expected format is 'RESET' or 'RESET key'. If you want to include special characters " + 
"in key, please use quotes, e.g., RESET `key`."
```
So no `s"""`, no `stripMargin` and no `replaceAll("\n", " ")`

Reply:
The `s` in `s"""` is not needed, as you point out. I believe the original author used `replaceAll` to avoid line breaks and make the message a single line.

Sure, I can change to using simple concatenation of two strings.